activemq-openwire-generator/pom.xml

```diff
@@ -44,9 +44,9 @@
       <artifactId>annogen</artifactId>
     </dependency>
     <dependency>
-      <groupId>ant</groupId>
+      <groupId>org.apache.ant</groupId>
       <artifactId>ant</artifactId>
-      <version>1.6.2</version>
+      <version>1.10.12</version>
```

Comment:
There are various other occurrences of Ant in the build per the previous discussion, which this change doesnt affect. Use dependencyManagement to align the dep versions from one place at the root.

Reply:
I don't see other direct other reference (outside of openwire generator). The only ant dependency I can see with `mvn dependency:tree` are transitive dependency. So I think it's OK like this.

Reply:
Yes they are transitive. No that doesnt make it OK. If the point is to upgrade to get CVEs fixes for example then they all need to be upgraded to actually achieve that. You wouldnt leave an old log4j 2.13 dep in place for example just because its transitive. The build should generally align on 1 version unless there is good reason not to.

Reply:
Hmm, ok. I just hope that the upgrade will work for the transitive. I'm trying and I will update the PR accordingly. Thanks.